### ISSUE TITLE
ubuntu lunar Release does not have a Release file.

### ISSUE THREAD

**Issue by SatoshiNakamotoBitcoin** [mynodebtc/mynode#753]
**Describe the bug**

sudo apt-get update
The repository 'http://ppa.launchpad.net/deadsnakes/ppa/ubuntu lunar Release' does not have a Release file.

**MyNode hardware (please complete the following information):**

 - Device: VM
 - Version v0.3.10 to v0.3.11

**Additional context**

sudo add-apt-repository --remove ppa:deadsnakes/ppa

Fixed.

**Comment by tehelsper** [comment]
How are you running mynode? Are you using Ubuntu somehow?

**Comment by SatoshiNakamotoBitcoin** [comment]
Yes, we think is only this Lunar Release Package, after that, all works fine.

Now we are in v0.3.11 and all ok.
